src/components/SendDialog.vue

```diff
@@ -251,8 +252,11 @@ export default defineComponent({
                     <q-card-section v-if="transactionId">
                         <div class="row">
                             <div class="col-12">
-                                <div class="row">You successfully sent {{ sendAmount }} {{ sendToken?.symbol }} to {{ receivingAccount }}.</div>
-                                <div class="row ellipsis-overflow" @click="navToTransaction">Click to view transaction: {{ transactionId }}</div>
+                                <div>You successfully sent {{ sendAmount }} {{ sendToken?.symbol }} to {{ receivingAccount }}.</div>
```

Comment:
I know that this is outside this issue's scope, but it feels a bit strange that the transaction result is not a standalone component. Is this by design or something that could be considered tech debt?

Reply:
yeah... i would say tech debt

Reply:
For reference, I created the issue https://github.com/telosnetwork/open-block-explorer/issues/659